packages/server/modules/fileuploads/tests/fileuploads.integration.spec.ts

```diff
@@ -46,8 +46,7 @@ describe('FileUploads @fileuploads', () => {
     ;({ token: userOneToken } = await createToken({
       userId: userOneId,
       name: 'test token',
-      scopes: [Scopes.Streams.Write],
-      lifespan: 3600
```

Comment:
hoping this fixes the test flakiness https://app.circleci.com/pipelines/github/specklesystems/speckle-server/12803/workflows/5ebc4ab4-e620-4a0e-b929-a1d6d43b27bf/jobs/199244/tests